models/staging/src_fivetran_platform.yml

```
@@ -200,5 +200,29 @@ sources:
             description: Phone number associated with user.
           - name: verified 
             description: Boolean that indicates whether the user has verified their email address in the account creation process.
+          - name: _fivetran_synced
+            description: Timestamp of when the record was last synced.
+
+      - name: transformation_runs
+        identifier: '{{ var("fivetran_log_transformation_runs_identifier", "transformation_runs") }}'
```

Small note - all other identifier variables have double quotes enclosing the bracket config and then single quotes for calling the variable. Let's use the same format as the other identifiers in this package to ensure consistency and that dbt doesn't behave any differently than expected across the board.
```suggestion
        identifier: "{{ var('fivetran_log_transformation_runs_identifier', 'transformation_runs') }}"
```